ocp_resources/resource.py

```diff
@@ -708,6 +708,10 @@ def wait_for_status(
                 self.logger.error(
                     f"Status of {self.kind} {self.name} is {current_status}"
                 )
+            else:
```

Comment:
This is not useful. if there is no current status, you already know that the status does not exist.

Reply:
removed